homeassistant/components/file/sensor.py

```diff
@@ -75,6 +86,7 @@ def __init__(
         self._file_path = file_path
         self._attr_native_unit_of_measurement = unit_of_measurement
         self._val_tpl = value_template
+        self._attr_unique_id = slugify(f"{name}_{file_path}")
```

Comment:
How do we know that name is unique? Maybe use the config entry id instead?

Reply:
We have a check when we import or add on `name` and `file_path`

Reply:
I don't think that is a good approach. The user should be able to name to whatever they want. That shouldn't be part of our integration logic.